UI/cypress/e2e/shortlist.feature

```diff
@@ -113,31 +113,31 @@ Feature: Tuition Partner shortlist
     Given a user has selected TPs to shortlist and journeyed forward to the shortlist page
     Then they choose to sort the shortlist by price
     And 'Zen Educate' is entry 1 on the shortlist page
-    And 'Reeson Education' is entry 2 on the shortlist page
-    And 'Tutors Green' is entry 3 on the shortlist page
-    And '3D Recruit Ltd' is entry 4 on the shortlist page
+    And 'Career Tree' is entry 2 on the shortlist page
```

Comment:
We are replacing the `3D Recruit Ltd` with the `Career Tree`. If it is, then can we please keep the ordering of the test cases as it is and replace the `3D Recruit Ltd` with the `Career Tree`.  i.e. we have here below previously
   ```
    And 'Reeson Education' is entry 2 on the shortlist page
    And 'Tutors Green' is entry 3 on the shortlist page
    And '3D Recruit Ltd' is entry 4 on the shortlist page
```
 with the new changes it should be the below 
   ```
 And 'Reeson Education' is entry 2 on the shortlist page
    And 'Tutors Green' is entry 3 on the shortlist page
    And 'Career Tree' is entry 4 on the shortlist page
```

Reply:
No, the issue is that these tests run off the live spreadsheets - so the data is live data and not mocked up.  So the tests need to be updated to use a new TP, which means that results are different.

Reply:
Okay, make sense.